tools/buildspec/cpu.yml

```diff
@@ -0,0 +1,38 @@
+version: 0.2
+
+env:
+  variables:
+    mxnet_variant: "cpu"
```

Comment:
You can specify the mxnet_variant using the console instead of the buildspec. Then, the same buildspec could be used for all variants.